app/controllers/ems_common.rb

```diff
@@ -813,6 +813,7 @@ def form_instance_vars
     @openstack_api_versions = retrieve_openstack_api_versions
     @vmware_cloud_api_versions = retrieve_vmware_cloud_api_versions
     @emstype_display = model.supported_types_and_descriptions_hash[@ems.emstype]
+    @vmware_cloud_amqp_security_protocols = retrieve_vmware_cloud_amqp_security_protocols
```

Comment:
Since this is AMQP, unrelated to the underlying provider, do you think we need to have provider-specific lists?